docs/import/manage_disaster.js

```diff
@@ -369,71 +370,67 @@ function addDisaster() {
  * Writes the given details to a new disaster entry in firestore. Fails if
  * there is an existing disaster with the same details.
  *
```

Comment:
maybe add comment detail about printing error since we removed from below?

Reply:
Not sure I understand. Saying here that this function now shows errors on failure? Added a couple of lines. Note that the prior comment, about not being the gd user, is no longer relevant since only the gd user can access this page.